x-pack/plugins/apm/public/components/app/mobile/transaction_overview/transaction_overview_tabs/stats_list/get_columns.tsx

```diff
@@ -0,0 +1,148 @@
+  comparisonEnabled?: boolean;
+  offset?: string;
+}): Array<ITableColumn<MobileMainStatisticsByFieldItem>> {
+  return [
+    // version/device
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.nameLabel',
+        {
+          defaultMessage: 'Name',
+        }
+      ),
+    },
+    // latency
+    {
+      field: 'latency',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.latencyColumnAvgLabel',
+        {
+          defaultMessage: 'Latency (avg.)',
+        }
+      ),
+      align: RIGHT_ALIGNMENT,
+      render: (_, { latency, name }) => {
+        const currentPeriodTimeseries =
+          detailedStatistics?.currentPeriod?.[name]?.latency;
+        const previousPeriodTimeseries =
+          detailedStatistics?.previousPeriod?.[name]?.latency;
+
+        const { currentPeriodColor, previousPeriodColor } = getTimeSeriesColor(
+          ChartType.LATENCY_AVG
+        );
+
+        return (
+          <SparkPlot
+            color={currentPeriodColor}
+            isLoading={detailedStatisticsLoading}
+            series={currentPeriodTimeseries}
+            valueLabel={asMillisecondDuration(latency)}
+            comparisonSeries={
+              comparisonEnabled && isTimeComparison(offset)
+                ? previousPeriodTimeseries
+                : undefined
+            }
+            comparisonSeriesColor={previousPeriodColor}
+          />
+        );
+      },
+    },
+    // throughput
+    {
+      field: 'throughput',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.throughputColumnAvgLabel',
+        { defaultMessage: 'Throughput' }
+      ),
+      align: RIGHT_ALIGNMENT,
+      render: (_, { throughput, name }) => {
+        const currentPeriodTimeseries =
+          detailedStatistics?.currentPeriod?.[name]?.throughput;
+        const previousPeriodTimeseries =
+          detailedStatistics?.previousPeriod?.[name]?.throughput;
+
+        const { currentPeriodColor, previousPeriodColor } = getTimeSeriesColor(
+          ChartType.THROUGHPUT
+        );
+
+        return (
+          <SparkPlot
+            color={currentPeriodColor}
+            isLoading={detailedStatisticsLoading}
+            series={currentPeriodTimeseries}
+            valueLabel={asTransactionRate(throughput)}
+            comparisonSeries={
+              comparisonEnabled && isTimeComparison(offset)
+                ? previousPeriodTimeseries
+                : undefined
+            }
+            comparisonSeriesColor={previousPeriodColor}
+          />
+        );
+      },
+    },
+    // crash rate
+    {
+      field: 'crashRate',
+      name: i18n.translate(
+        'xpack.apm.mobile.transactions.overview.table.crashRate',
+        {
+          defaultMessage: 'Crash rate',
+        }
+      ),
+      align: RIGHT_ALIGNMENT,
+      render: (_, { crashRate }) => {
```

Comment:
I realize this is out of scope, but why don't we have a spark plot/comparison for crash rate?

Reply:
I followed the designs, but I will take a note and ask @boriskirov when he is back